featuretools/feature_base/feature_base.py

```diff
@@ -82,19 +86,14 @@ def get_name(self):
             self._name = self.generate_name()
         return self._name
 
-    def get_names(self):
+    def get_feature_names(self):
```

Comment:
we should update `test_serialized_renamed_features` in `test_feature_serialization.py` to test `get_feature_names` as well

The test case also appears to be missing tests for a multi-output feature, and a FeatureOutputSlice feature